controlplane/cmd/nsmd/main.go

```diff
@@ -14,6 +16,12 @@ import (
 
 var version string
 
+// Default values and environment variables of proxy connection
+const (
+	NsmdAPIAddressEnv      = "NSMD_API_ADDRESS"
+	NsmdAPIAddressDefaults = "0.0.0.0:5001"
```

Comment:
Could we make this a list, and default to listening on the wildcard v6 address as well?

Reply:
I changed default API Address to ":5001" so NSM should listen on both ipv6 and ipv4 now